news/_posts/releases/2020-11-18-prestashop-1-7-7-0-available.md

```diff
@@ -0,0 +1,122 @@
+---
+layout: post
+title:  "PrestaShop 1.7.7.0 is available"
+subtitle: "You waited for it, here it is!"
+date:   2020-11-18 16:00:00
+authors: [ PrestaShop ]
+image: /assets/images/2020/11/build_and_mail_final.jpg
+icon: icon-leaf
+tags:
+ - version
+ - 1.7
+ - 1.7.7.x
+ - minor
+ - releases
+---
+
+We are happy to announce PrestaShop 1.7.7.0 is officially available!
+
+![1.7.7.0 is available!](/assets/images/2020/11/build_and_mail_final.jpg)
+
+This release is our biggest yet since 1.7.1.0. More than one year of development, two betas, one release candidate Was it worth it? We absolutely think so, and hope that you will enjoy using it.
+
+## News in the 1.7.7.0
+
+### Productivity
+
+The **[full redesign and the new features of order pages]**(https://build.prestashop.com/news/the-new-order-pages-in-prestaShop-1-7-7/) help merchants to make a better use of their time and to stay focus on their online business. 
+
+The redesign of the user interface allows merchants to find the right information quickly while the new features allow a better efficiency on daily tasks and avoid back and forth between pages. 
+
+### Growth and flexibility
+
+PrestaShop is an international solution and we aim at providing highly global features for worldwide users. 
+
+The **[international improvements]**(https://build.prestashop.com/news/prestashop-keeps-growing-internationally/) remove barriers to go international and help merchants reach more customers.
```

Comment:
```suggestion
The **[international improvements]**(https://build.prestashop.com/news/prestashop-keeps-growing-internationally/) widen once again the PrestaShop frontier allowing merchants to go international and reach more customers.
```

Reply:
```suggestion
The **[international improvements]**(https://build.prestashop.com/news/prestashop-keeps-growing-internationally/) remove barriers for merchants who want to expand internationally and reach more customers.
```
I am not a big fan of "PrestaShop frontier" as we are international, here is my suggestion @MatShir